Content.Server/GameTicking/Rules/TraitorRuleSystem.cs

```diff
@@ -97,6 +98,14 @@ public bool MakeTraitor(EntityUid traitor, TraitorRuleComponent component, bool
 
         _antag.SendBriefing(traitor, GenerateBriefing(component.Codewords, code, issuer), null, component.GreetSoundNotification);
 
+        // Send codewords to only the traitor client
+        var color = Color.FromHex("#8f4a4b"); // Falls back to a dark red Syndicate color if a prototype is not found
+        if (_prototypeManager.TryIndex("Syndicate", out RadioChannelPrototype? syndieChannel))
+        {
+            color = syndieChannel.Color;
+        }
+        RaiseNetworkEvent(new RoleCodewordEvent(GetNetEntity(traitor), color, "traitor", component.Codewords.ToList()), traitor);
```

Comment:
Since roles and assigned objectives are attached to the player's mind rather than their character, it would probably be better to do the same with the codewords.